src/app/dataset/actionbar/actionbar.component.spec.ts

```diff
@@ -77,8 +77,8 @@ describe('ActionbarComponent', () => {
     fixture.detectChanges();
     expect(workflows.promptCancelThisWorkflow).toHaveBeenCalledWith(
       '253453453',
-      undefined,
-      undefined
+      (undefined as unknown) as string,
+      (undefined as unknown) as string
```

Comment:
This seems magic :) What does it do?

Reply:
This was to satisy the compiler with its latest libraries, as restoring it to return "undefined" (x2) results in the error:

ERROR in src/app/dataset/actionbar/actionbar.component.spec.ts(80,7): error TS2345: Argument of type 'undefined' is not assignable to parameter of type 'string | AsymmetricMatcher<any>'.